src/test/java/reactor/core/publisher/TopicProcessorTest.java

```diff
@@ -414,4 +428,212 @@ public void customRequestTaskThreadRejectsNull() {
 		          );
 	}
 
+	@Test
```

Comment:
I would consider breaking this one into as many tests as `processor = ` blocks, even if they have nondescript names like `factoryMethod12`, the test is really too big even for my taste :p